src/context.ts

```diff
@@ -262,6 +268,22 @@ export class Context<E extends WebhookPayloadWithRepository = any>
     ) as unknown) as T;
   }
 
+  /**
+   * @see config
+   *
+   * @param fileName - Name of the YAML file in the `.github` directory
+   * @param defaultConfig - An object of default config options
+   * @param deepMergeOptions - Controls merging configs (from the [deepmerge](https://github.com/TehShrike/deepmerge) module)
+   * @return Configuration object read from the file
+   */
+  public configWithoutBase<T>(
```

Comment:
not sure about this shortcut or the name of it.

`configWithoutBase` may be confusing for configs that actually reference base (and in that case we still load the base)